app/parse.py

```diff
@@ -8,8 +15,49 @@ class Quote:
     tags: list[str]
 
 
+def get_info(soup):
+    text = soup.select_one(".text").text
+    author = soup.select_one(".author").text
+    tags = [
+        tag.text for tag in
+        soup.select(".tag")
+    ]
+    info = {
+        "text": text,
+        "author": author,
+        "tags": tags
+    }
+    QCl = Quote(**info)
+    return f"{QCl.text},{QCl.author},{QCl.tags}"
```

Comment:
better return `Quote` object here and use data from it when you write to csv file